### DISCUSSION TITLE
Add show action for all modules to print and register attributes if needed

### DISCUSSION THREAD

**Discussion by rmb82** [freeipa/ansible-freeipa#1293] [Ideas]
It's possible to add "show' action to print all raw (or not) attributes of user, group, hbacrule, sudorule, idview ...etc ?
To register attributes needed and reuse them.

as :
- ipa group-show  group_name --raw
- ipa user-show  user --raw

Thx

**Comment by t-woerner** [comment]
There is an older draft for the query mode:  https://github.com/freeipa/ansible-freeipa/pull/782
This needs to be adapted and extended.
